cms/djangoapps/contentstore/courseware_index.py

```diff
@@ -175,8 +181,31 @@ def index_item(item, skip_index=False, groups_usage_info=None):
                 return
 
             item_content_groups = None
+
+            if item.category == "split_test":
+                user_partitions = item.user_partitions
+                split_partitions = []
+                if user_partitions:
+                    for user_partition in user_partitions:
+                        if user_partition.scheme.name == RANDOM_SCHEME:
+                            split_partitions.append(user_partition)
+                for vertical in item.get_children():
+                    if split_partitions:
+                        for user_partition in split_partitions:
+                            for group in user_partition.groups:
+                                group_id = unicode(group.id)
+                                child_location = item.group_id_to_child.get(group_id, None)
```

Comment:
Would result in the same error message I already mentioned in my first comment.
Using anything from `split_test_module.py` simply won't work.